Sources/SwiftFusion/Geometry/Pose2.swift

```diff
@@ -28,7 +28,7 @@ public struct Pose2: Equatable, Differentiable, KeyPathIterable, TangentStandard
 
 @differentiable
 public func inverse(_ p: Pose2) -> Pose2 {
-  Pose2(inverse(p.rot_), p.rot_.unrotate(-p.t_))
+  Pose2(inverse(p.rot_), p.rot_.unrotated(-p.t_))
```

Comment:
I think this should be `unrotate` because `p.rot_` is not modified.

Reply:
Agreed, plus matches GTSAM